keyboards/gopolar/gg86/keymaps/horrortroll/led/cool_diagonal.c

```diff
@@ -0,0 +1,6 @@
+static HSV COOL_DIAGONAL_math(HSV hsv, uint8_t i, uint8_t time) {
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

Reply:
Done